exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -248,6 +248,12 @@ func newLogsExporter(config Config, set exporter.Settings, marshalers map[string
 		return nil, errUnrecognizedEncoding
 	}
 
+	if config.PartitionLogsByResourceAttributes {
```

Comment:
Else error or warn?

Reply:
Can you elaborate a bit more?  

This if guards a configurable behavior, so if the configuration isn't set are you suggesting we log that out at this point?

Reply:
sorry wrong line, let me open at the right line